thirdeye/thirdeye-frontend/app/pods/components/rootcause-data-indicator/component.js

```diff
@@ -39,7 +39,12 @@ export default Component.extend({
 
     fetch(`/rootcause/raw?framework=identity&urns=${selectedUrn}`)
       .then(checkStatus)
-      .then(res => setProperties(this, { maxTime: parseInt(res[0].attributes.maxTime[0], 10) }));
+      .then(res => {
```

Comment:
Not a fan of using super short variable names like `res` vs something clearer like `result` even if it is more concise. It's like coming in and being all "what is res? *stare* ooooh result"